src/main/java/uk/gov/hmcts/ccd/endpoint/ui/QueryEndpoint.java

```diff
@@ -164,7 +167,8 @@ public SearchResultView searchNew(@PathVariable("jid") final String jurisdiction
                                       @PathVariable("ctid") final String caseTypeId,
                                       @RequestParam java.util.Map<String, String> params) {
         final String view = params.get("view");
-        MetaData metadata = new MetaData(caseTypeId, jurisdictionId);
+        String validCaseTypeId = validateCaseTypeId(caseTypeId);
```

Comment:
Validation logic should always be the very first line unless a strong rationale necessitates otherwise. Also, you don't need to work with a new validCaseTypeId variable. Just the below line could read more smooth?
```
validateCaseTypeId(caseTypeId);
```